solidity/contracts/test/ReceiveApprovalStub.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity <0.9.0;
+
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
```

Comment:
I think we can remove this import

Reply:
🔥 in https://github.com/keep-network/tbtc-v2/pull/5/commits/d671d1a73c27b5b2dce30126cab6b12a9d873db3